public/js/p3/widget/app/templates/Homology.html

```diff
@@ -12,14 +12,11 @@ <h3>BLAST
                             title="click to open tutorial"></i></a>
                 </div>
             </h3>
-            <p>The BLAST service integrates the BLAST (Basic Local Aligment Search Tool) algorithms to perform
-                searches against public or private genomes or other reference databases using DNA or protein
-                sequence(s).
-                For further explanation, please see BLAST
-                Service <a href="${docsServiceURL}${applicationHelp}" target="_blank">Quick Reference Guide</a> and
-                <a href="${docsServiceURL}${tutorialLink}" target="_blank">Tutorial</a>.
-            </p>
+            <p>${applicationDescription} For further explanation, please see the ${applicationLabel} Service <a
+                    href="${docsServiceURL}${applicationHelp}" target="_blank">Quick Reference Guide</a> and <a
+                    href="${docsServiceURL}${tutorialLink}" target="_blank">Tutorial</a>.</p>
         </div>
+        <link rel="stylesheet" href="/js/p3/resources/p3.css">
```

Comment:
This can't be here but am curious what the styling issue was that required it.

Reply:
There is inconsistency in the styling between services. Some services use mixed case font such as BLAST while others use upper case font such as the MSA service. I was trying to figure out how to change the styling. I removed this line in a commit.